File: app/src/main/java/com/tebet/mojual/view/saledetail/SaleDetailFragmentProvider.java
package com.tebet.mojual.view.saledetail;

import dagger.Module;
import dagger.android.ContributesAndroidInjector;

@Module
public abstract class SaleDetailFragmentProvider {

    @ContributesAndroidInjector(modules = SaleDetailFragmentModule.class)
    abstract SaleDetailFragment provideSaleDetailFragmentFactory();
}

File: app/src/main/java/com/tebet/mojual/view/saledetail/SaleDetailFragmentProvider.kt
package com.tebet.mojual.view.saledetail

import dagger.Module
import dagger.android.ContributesAndroidInjector

@Module
abstract class SaleDetailFragmentProvider {

    @ContributesAndroidInjector(modules = [SaleDetailFragmentModule::class])
    internal abstract fun provideSaleDetailFragmentFactory(): SaleDetailFragment
}
